SpeedyTransforms/ext/SpeedyTransformsAMDGPUExt.jl

```diff
@@ -0,0 +1,92 @@
+module SpeedyTransformsAMDGPUExt
```

Comment:
Isn't this extension essentially identical to the SpeedyTransformsCUDAExt? If yes we could move the generic code dispatched over `GPUArrays.AbstractGPUArray` into SpeedyTransforms directly and let the extension only define `which_FFT_package` ?